pool/src/lib.rs

```diff
@@ -474,7 +501,7 @@ impl LendingPoolTrait for LendingPool {
         }
 
         // amount_to_burn = amount_to_withdraw / liquidity_index
-        let amount_to_burn = Self::get_collateral_coeff(&env, &reserve)?
+        let amount_to_burn = Self::get_collat_coeff(&env, &asset, &reserve)?
```

Comment:
We need use `amount_to_burn` instead of `amount_to_withdraw` above

Reply:
Do we need to pass compunded balance instead of who_balance in validate_withdraw?